src/GraphQL/Utilities/Visitors/AppliedDirectivesValidationVisitor.cs

```diff
@@ -7,23 +7,31 @@ namespace GraphQL.Utilities
     /// <summary>
     /// This visitor verifies the correct application of directives to the schema elements.
     /// </summary>
-    internal sealed class AppliedDirectivesValidationVisitor : ISchemaNodeVisitor
+    public sealed class AppliedDirectivesValidationVisitor : ISchemaNodeVisitor
```

Comment:
So... now you need to add `inheritdoc `everywhere.

Reply:
It doesn't bother me.  It would be nice if .NET did not require inheritdoc when it already infers the doc inheritance.